ExampleApp/ExampleApp/Screens/ScanResults/ViewModel/CellViewModelItem/ScanResultViewModelItem.swift

```diff
@@ -0,0 +1,38 @@
+import RxBluetoothKit
+import UIKit
+
+final class ScanResultsViewModelItem: SectionModelItem {
+
+    var rowData: [Any] {
+        return peripheralRowItems
+    }
+
+    var itemsCount: Int {
+        return peripheralRowItems.count
+    }
+
+    var sectionName: String?
+
+    private(set) var peripheralRowItems: [ScannedPeripheral]
+
+    init(_ sectionName: String, peripheralRowItems: [ScannedPeripheral] = []) {
+        self.sectionName = sectionName
+        self.peripheralRowItems = peripheralRowItems
+    }
+
+    func cellClass() -> UIView.Type {
+        return ScanResultTableViewCell.self
+    }
+
+    func append(_ item: Any) {
+        if let item = item as? ScannedPeripheral {
```

Comment:
I think `guard` would starve better here. Also, no need for all of it if types are present :)